### ISSUE TITLE
Set cancel button at bottom.

### ISSUE THREAD

**Issue by ChrisOSX** [alexanderjarvis/PXAlertView#46]
How would one set the cancel button at the bottom of the alert?

**Comment by alexanderjarvis** [comment]
I don't think this is default behaviour in UIAlertView (as an example). Perhaps only if you use the addButton api instead of creating all the buttons during init.
